_data/gallery.yml

```diff
@@ -1,3 +1,92 @@
+  authors:  "Xinya Du, Bhavana Dalvi Mishra, Niket Tandon, Antoine Bosselut, Wen-tau Yih, Peter Clark, Claire Cardie"
+  venue:    "NAACL 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Be-Consistent-!-Improving-Procedural-Text-using-Du-Mishra/6a4b27f8bde3e8cd65f4d9633339f7149b878672"
+  - code:   "https://github.com/allenai/propara"
+
+###############################################################################
+- title:    "Structural Scaffolds for Citation Intent Classification in Scientific Publications"
+  authors:  "Arman Cohan, Waleed Ammar, Madeleine van Zuylen, Field Cady"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Structural-Scaffolds-for-Citation-Intent-in-Cohan-Ammar/99dd70ad75af11ce84098f58e5a9ae522cc73e3b"
+  - code:   "https://github.com/allenai/scicite"
+
+###############################################################################
+- title:    "Combining Distant and Direct Supervision for Neural Relation Extraction"
+  authors:  "Iz Beltagy, Kyle Lo, Waleed Ammar"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Combining-Distant-and-Direct-Supervision-for-Neural-Beltagy-Lo/8ed76934cf262f38aa9fa0d937c2dcd4d2673684"
+  - code:   "https://github.com/allenai/comb_dist_direct_relex/"
+
+###############################################################################
+- title:    "Still a Pain in the Neck:Evaluating Text Representations on Lexical Composition"
+  authors:  "Vered Shwartz, Ido Dagan"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Still-a-Pain-in-the-Neck%3A-Evaluating-Text-on-Shwartz-Dagan/1b7613d21b074d7bc1bdb868a5b46751b2064d17"
+  - code:   "https://github.com/vered1986/lexcomp"
+
+###############################################################################
+- title:    "Zoho at SemEval-2019 Task 9: Semi-supervised Domain Adaptation using Tri-training for Suggestion Mining"
+  authors:  "S. R. Mahadeva Prasanna, Sri Ananda Seelan"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Zoho-at-SemEval-2019-Task-9%3A-Semi-supervised-Domain-Prasanna-Seelan/05ae1917294570cc42e3d39799720bb4fc8d3d93"
+  - code:   "https://github.com/sai-prasanna/suggestion-mining-semeval19"
+
+###############################################################################
+- title:    "75 Languages, 1 Model: Parsing Universal Dependencies Universally"
+  authors:  "Daniel Kondratyuk"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/75-Languages%2C-1-Model%3A-Parsing-Universal-Kondratyuk/6a2daaa16306c8b06d7711e55ffe9ca08dd68c7a"
+  - code:   "https://github.com/hyperparticle/udify"
+
+###############################################################################
+- title:    "DROP: A Reading Comprehension Benchmark Requiring Discrete Reasoning Over Paragraphs"
```

Comment:
This one is in the main repo; not sure why it should be here.

Reply:
I removed DROP.  I would include models that are in the main repo if they are contributed.